packages/components/src/DatePicker/DatePickerActivator.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { DatePickerActivator } from "./DatePickerActivator";
+import { Button } from "../Button";
+
+it("renders a button by default", () => {
+  const { container } = render(<DatePickerActivator />);
+  expect(container.querySelector("Button")).toBeInTheDocument();
```

Comment:
I'd use the `getBy` instead of query selector. Keep it pure jest things when testing jest until otherwise you can't.

Reply:
also test if the calendar icon is there as well

Reply:
https://github.com/GetJobber/atlantis/pull/783/commits/42f4d9a8bbd0e407ec9979c9ecd64907647ed958